Library/Formula/macvim.rb

```diff
@@ -1,12 +1,10 @@
-# Reference: https://github.com/b4winckler/macvim/wiki/building
```

Comment:
Sorry, just noticed: is this not just better redirected to https://github.com/macvim-dev/macvim/wiki/building now?